DataFormats/HeavyIonEvent/src/HFFilterStruct.cc

```diff
@@ -0,0 +1 @@
+#include "DataFormats/HeavyIonEvent/interface/HFFilterStruct.h"
```

Comment:
is this needed?

Reply:
Actually is not needed, I included that to be consistent with the other codes in DataFormats/HeavyIonEvent/ , but I can remove if you want.

Reply:
I have removed the .cc in commit 78b0a76436417538aefafc8a492dd0314831b794